security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml

```diff
@@ -1181,4 +1181,19 @@
         <label>KAS Authtype</label>
         <type>dropdown</type>
     </field>
+    <field>
+        <label>deSEC.io API</label>
+        <type>header</type>
+        <style>table_dns table_dns_dedyn</style>
```

Comment:
The acme.sh dns hook is "dns_desec", all fields should use this name instead of "dns_dedyn".

Reply:
Thanks for review! It is fixed now.